config/initializers/lang.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+LANG = Rails.application.config_for(:lang)
+
+Hyacinth::Config.load_default_subtags! if ActiveRecord::Base.connection.table_exists? :language_subtags
```

Comment:
So to verify, this is supposed to run every time the app starts in case someone changed the default subtags since the last application restart?

Reply:
Yes - and we don't plan on storing default tag data on the values.